### DISCUSSION TITLE
iped.exe: "This app can't run on your PC" error

### DISCUSSION THREAD

**Discussion by gilmaralves** [sepinf-inc/IPED#674] [Q&A]
After downloaded the last two releases and run the comand "iped.exe -d relatorio.ufdr -o teste", I received the message "this app can't run on your PC".

The PC is a Dell machine running Windows 10 Pro and Java 1.8.0_301.

Does anyone have a tip to solve the problem?

**Comment by lfcnassif** [comment]
What is your OS architecture 32bits or 64bits? Try to use "java -jar iped.jar" instead of "iped.exe"

**Reply by gilmaralves** [reply]
The OS architecture is 64 bits; I tried "java -jar iped.jar" successfully
